### PR TITLE
favicon relative path

### LINKED ISSUE #139: Powered by CATS pic on Careers Page does not appear correctly
Hi This one is a simpler one for you guys I guess...

The 'Powered by CATS' pic that is supposed to appear on Careers page at the footer seems to me missing from the link and is thus appearing in poor way on the page. Not being a Techie I am not able to figure out where to change that link picture...

Can you please let me know the file and the link I am supposed to put in there so that it appears correctly in the Page footer...???/

### PR DESCRIPTION
**Pull request by filippo-basso**
Hi,
this is a very simple test for a pull request, to see that I do it cleanly and without any mistake.
Not so important, but the favicon was on an absolute path, so not found when OpenCATS is in a folder and not installed at root folder.

I've some questions, but I hope I can ask you in slack.
Fi

resolves #139

### CODE REVIEW

**Comment by skrchnavy** [comment]
relates #139

**Comment by filippo-basso** [comment]
Thank you skrchnavy,
I haven't even searched for /images, but it's better like this!
Fi

File: modules/careers/Blank.tpl [hunk 1]
```diff
@@ -37,7 +37,7 @@
 
         <?php /* WARNING: It is against the terms of the CPL to remove or alter the following line.  The 'Powered by CATS' line must stay visible on every page. */ ?>
         <div id="poweredCATS">
-		<a href="http://www.catsone.com" target="_blank"><img src="/images/CATS-powered.gif" alt="Powered by: CATS - Applicant Tracking System" title="Powered by: CATS - Applicant Tracking System" /></a>
+		<a href="http://www.catsone.com" target="_blank"><img src="images/CATS-powered.gif" alt="Powered by: CATS - Applicant Tracking System" title="Powered by: CATS - Applicant Tracking System" /></a>
```
Comment (skrchnavy) [inline modules/careers/Blank.tpl]: @filippo-basso please correct this row, the image is in opencats root not in career portal part.

**Comment by RussH** [comment]
Approved - merged!

**Pull request merged**